idm/.env.template

```diff
@@ -5,5 +5,5 @@ GITHUB_CLIENT_ID=<from above>
 GITHUB_CLIENT_SECRET=<from above>
 APP_BASE_URL=http://idm.learnersguild.dev  # both of these
 IDM_BASE_URL=http://idm.learnersguild.dev  # are needed
-JWT_PRIVATE_KEY="<quoted string data from /tmp/private-key.pem with \n for newlines>"
-JWT_PUBLIC_KEY="<quoted string data from /tmp/public-key.pem with \n for newlines>"
\ No newline at end of file
+JWT_PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----\nMIIEowIBAAKCAQEAuV9BPSAt4H02mbTq9dwfAw2C7QNrBRv1RPadjMD9chkrY7E2\nnf9K+PEWygJAyBm02AnD5hITmQJYyT1Fb8AJuZWUp6rctmWb+3hg+2mXvhfx1qvn\nYlagJKk5cC/Tw0j8VBicVbwt30Ya7dPrlKgdASjg3/dUhPs5Q7VmZACDn6GT1a/f\nk8TUSbgAUMwrpuRh6g1wk2BP/tQWQZPgJciVtfNLhdz/pIWp3Ke+U1iidtChgfpu\nR62VGhTP1N/O4yKaIq1v2ttAjbESkjrA+YieIyHwUvtEoewkU12llxZwPOrGb7r4\nPDARf7GaU20EEuRE0OuWqYIbVe+58XbrszPWwwIDAQABAoIBADoL8/faHcOQMgrL\ntk+q0CM6leb2DQsaiCFCQG+U90hFTWr4USwEh0uQ4xv6vKJsOk70L93Zhz2AmKDb\niOTHc8QjbNtcC6UqCEAkn+naVOP3x4j7Z4dJrPgN7aiNsXgTYMBMvJe0R4EFCOW0\nWK7CPeLhFBnN7y99I59qLFfgOVdAztNL1gVbJ10XALvhaqovsUgRXiQjOv+QUCw4\nw17WQ/4m75SSaGMMnW0hi7KIibj6lNhHDx3UZLxMGIX2/FTqJQdMt0pTwCcY2pvS\nqnevCdv3WNU1sU37VaBg+4EbdJhg+q2wBJDDdLbv4HCZSPZRBFdIrJs5ezREqVy5\nLDYcmRECgYEA5eYLP+fAy/PZO0AmRNGP2UqVucLA6E7qNGUKk1qehN/K2FRVlDUu\nO2UO0DTaYd9iZR2FXSt+rqYaYOrRVHcrxRuXqambaVTvVjwBkXMeIjWs7IxkOPn/\nJQphPQ+LKsPzt2CQxZekhcf6c6w0tHRn5QSpra8MJt94ZHAsz85Rz+8CgYEAzmsO\ng90zQHE8B93zPHzVDUEA1I9jqdEN4wCgKH9/wTHVWUccILHq34nZCc+zqiGRimvx\nccAz9TBub2M213ttMKvzPNYGANnJ+PQ7wHLP9Oq4/YgWLapQGU9wq92BeF7GK734\n/t06hLogyskg/USYbvaId3zEdqyByswUVSK4km0CgYAqejfH5f00cejITtEDVCkc\n/+/sM7TK95HEforWsR6q9GKZV/O7/5t8M+7pv2o+PTUD89FpZpjwkhY7pAdrcN1w\nfOmn2oOxkZTrPQkZI8kEjyBVUJdQA7kF0gYnbxDKpQwJ7Niy+ZLXeRyodMAEO6Ha\nqeLkCXzYTeADxsow1zE8lwKBgFFbwunYesXwm6+iHKf/r928fD5LkgWx315Utcug\njVMxeBzrOTvtxFx9Ma17Tx5Egt/jMdQa5J/EihKfMVN3U482hvW4ib82GDRm11JA\nLpUjGuzbAZrYISAIvqkO8mPmPweopgaWXeVIk3o6EiWhLs+1ng4ygCEiVzn+fBke\nSU91AoGBAKQ7wJ4IP+VshDIPeVUIK5NZtunTsS+JKoey+RUzyc14cz7m1rK0K51i\ncc1c3Ewt3P4Uhr3g5imEoRo0KWJEqHWofrmIyoRX6G+2+DheHu+EN4f6WlxCeTiG\nYjjZuXNdxqGYp5bR9LVc2HYG9HRCjoYjPyssi2YLmWChxGRRaFKs\n-----END RSA PRIVATE KEY-----"
```

Comment:
changed back, I altered it during my weird bug of not being able to log into IDM on accident.